src/main/resources/db/migration/V5.11.0.1__8605-support-archival-status.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE datasetversion SET archivalCopyLocation = CONCAT('{"status":"success", "message":"', archivalCopyLocation,'"}') where archivalCopyLocation is not null and not archivalCopyLocation='Attempted';
```

Comment:
If this script is only needed by TDL as suggested in the PR description, perhaps we don't need it.

Reply:
`UPDATE datasetversion SET archivalCopyLocation = CONCAT('{"status":"success", "message":"', archivalCopyLocation,'"}') where archivalCopyLocation is not null` is needed for standard instances (those that have used archiving and therefore have non-null entries). The `and not archivalCopyLocation='Attempted';` and the second line handle the case that TDL deployed which was in the initial PR #8610 which has gotten passed by this PR.

Reply:
Ok, I guess my understanding is that both lines are needed or at least won't hurt anything.